sdk/python/feast/infra/utils/aws_utils.py

```diff
@@ -50,6 +50,10 @@ def get_bucket_and_key(s3_path: str) -> Tuple[str, str]:
     return bucket, key
 
 
+@retry(
+    wait=wait_exponential(multiplier=1, max=30),
```

Comment:
We probably should reduce the max here. This would do 2^30 seconds as a max I think. We probably don't really care beyond max=3 or max=4

Reply:
Done